galaxy_ng/app/tasks/deletion.py

```diff
@@ -0,0 +1,36 @@
+import logging
+
+from pulpcore.app.tasks import orphan_cleanup
```

Comment:
this will need to be moved to pulpcore.plugin for CI to pass

Reply:
@awcrosby you can add here https://github.com/ansible/galaxy_ng/blob/master/template_config.yml#L28 and regenerate ci template

Reply:
oh great, i'll do that

Reply:
anyway I think it is a good idea to add it to pulpcore plugin